test/python/pulse/test_parameter_manager.py

```diff
@@ -14,491 +14,12 @@
-
-        value_dict = {self.amp1_1: 0.1, self.amp1_2: 0.2, self.dur1: 160}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_obj = pulse.Gaussian(duration=160, amp=0.3, sigma=40)
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_set_parameter_to_acquire(self):
-        """Test set parameters to acquire instruction."""
-        test_obj = pulse.Acquire(16000, pulse.AcquireChannel(self.ch1), pulse.MemorySlot(self.ch1))
-
-        value_dict = {self.ch1: 2}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_obj = pulse.Acquire(16000, pulse.AcquireChannel(2), pulse.MemorySlot(2))
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_set_parameter_to_inst(self):
-        """Test get parameters from instruction."""
-        test_obj = pulse.ShiftPhase(self.phi1 + self.phi2, pulse.DriveChannel(0))
-
-        value_dict = {self.phi1: 0.123, self.phi2: 0.456}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_obj = pulse.ShiftPhase(0.579, pulse.DriveChannel(0))
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_set_parameter_to_call(self):
-        """Test get parameters from instruction."""
-        sched = pulse.Schedule()
-        sched += pulse.ShiftPhase(self.phi1, self.d1)
-
-        with self.assertWarns(DeprecationWarning):
-            test_obj = pulse.Call(subroutine=sched)
-
-        value_dict = {self.phi1: 1.57, self.ch1: 2}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_sched = pulse.Schedule()
-        ref_sched += pulse.ShiftPhase(1.57, pulse.DriveChannel(2))
-
-        with self.assertWarns(DeprecationWarning):
-            ref_obj = pulse.Call(subroutine=ref_sched)
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_with_function(self):
-        """Test ParameterExpressions formed trivially in a function."""
-
-        def get_shift(variable):
-            return variable - 1
-
-        test_obj = pulse.ShiftPhase(get_shift(self.phi1), self.d1)
-
-        value_dict = {self.phi1: 2.0, self.ch1: 2}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_obj = pulse.ShiftPhase(1.0, pulse.DriveChannel(2))
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_set_parameter_to_alignment_context(self):
-        """Test get parameters from alignment context."""
-        test_obj = AlignEquispaced(duration=self.context_dur + self.dur1)
-
-        value_dict = {self.context_dur: 1000, self.dur1: 100}
-
-        visitor = ParameterSetter(param_map=value_dict)
-        assigned = visitor.visit(test_obj)
-
-        ref_obj = AlignEquispaced(duration=1100)
-
-        self.assertEqual(assigned, ref_obj)
-
-    def test_nested_assignment_partial_bind(self):
```

Comment:
Please keep this test. You can just remove dependency on Call and add subroutine directly here because subroutine is ScheduleBlock.